digitalocean/database/datasource_database_ca_test.go

```diff
@@ -54,7 +54,7 @@ func TestAccDataSourceDigitalOceanDatabaseCA(t *testing.T) {
 							}
 							cert, err := x509.ParseCertificate(block.Bytes)
 							if err != nil {
-								return fmt.Errorf("failed to parse certificate: " + err.Error())
+								return fmt.Errorf("failed to parse certificate: %s", err.Error())
```

Comment:
I'm not really sure what the specifier should be. Maybe `%v` or `%w` instead of `%s`. Anyway, it's not allowed to use dynamic/non-static variables in error-messages, without a format-specification as of Go 1.24+